products/cpdb/bash/01_dataloading.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
 source bash/config.sh
 
+# TODO mimic template.load
 # Reference tables
-run_sql_file "sql/_create.sql"
+echo "Dropping and creating build schema '$BUILD_ENGINE_SCHEMA'"
+# create build schema
+run_sql_command \
+    "
+    DROP SCHEMA IF EXISTS ${BUILD_ENGINE_SCHEMA} CASCADE;
+    CREATE SCHEMA ${BUILD_ENGINE_SCHEMA};
+    "
 create_source_data_table
```

Comment:
on our journey from bash to python, this could now be done via `dcpy.builds.enviroment`

that would help make future orchestration cleaner right?